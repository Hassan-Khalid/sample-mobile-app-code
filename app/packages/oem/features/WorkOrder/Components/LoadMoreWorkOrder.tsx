import React from 'react';
import { ActivityIndicator, View } from 'react-native';

function LoadMoreWorkOrder() {
  return (
    <View style={{ height: 100, width: "100%" }}>{<ActivityIndicator />}</View>
  );
}

export default LoadMoreWorkOrder;
