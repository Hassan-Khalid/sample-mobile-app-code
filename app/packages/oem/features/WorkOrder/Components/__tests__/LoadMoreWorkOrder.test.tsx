import { render } from '@testing-library/react-native';
import LoadMoreWorkOrder from '../LoadMoreWorkOrder';
import React from 'react';

describe('Load More WorkOrder',()=>{
  it('render component', () => {
    //setup
    const wrapper = render(<LoadMoreWorkOrder />);
    
    // post-conditions
    expect(wrapper.toJSON).toMatchSnapshot();
  });
  
});

