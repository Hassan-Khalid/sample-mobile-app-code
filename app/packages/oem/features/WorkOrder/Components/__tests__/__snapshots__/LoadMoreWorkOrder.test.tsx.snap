// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Load More WorkOrder render component 1`] = `[Function]`;
